azavea/cac-tripplanner
# Switch to Chrome Headless for running JS tests

## Overview

PhantomJS is deprecated. Use Chrome Headless instead.


### Demo

![image](https://user-images.githubusercontent.com/960264/62381859-63d21400-b51a-11e9-9963-139b8fff7baa.png)


## Testing Instructions

 * `vagrant provision app`
 * `vagrant ssh app && cd /opt/app/src && npm run gulp-test`
 * Tests should pass and use Chrome Headless
